app/config/av-scan.js

```diff
@@ -0,0 +1,35 @@
+const Joi = require('joi')
```

Comment:
Remove everything related to the av-scan as we will be using a different setup now